src/node/rpc/frontend.h

```diff
@@ -592,7 +610,7 @@ namespace ccf
            ctx.frame_format()});
       };
 
-      auto rep = process_command(ctx, tx, fn);
+      auto rep = process_command(ctx, tx, fn, last_idx, max_conflict_version);
```

Comment:
This is sometimes `last_idx` and sometimes `reserved` - these don't sound like the same property.

Reply:
fix the naming